### PR TITLE
support srcset

### PR DESCRIPTION
**Pull request by MillerRen**
support srcset.

### CODE REVIEW

File: vue-lazyload.es5.js [hunk 1]
```diff
@@ -135,7 +135,12 @@ var vueLazyload = (function (Vue) {
         if (!bindType) {
             el.setAttribute('src', src);
         } else {
-            el.style[bindType] = 'url(' + src + ')';
+            // support srcset attribute e.g v-lazy:srcset="src". ugly but worked fine :)
```
Comment (darrynten) [inline vue-lazyload.es5.js]: You need to do these changes in both the es5 and the es6 versions of the script

File: vue-lazyload.es5.js [hunk 2]
```diff
@@ -135,7 +135,12 @@ var vueLazyload = (function (Vue) {
...
+            if (!bindType.indexOf('-') > -1) {
```
Comment (darrynten) [inline vue-lazyload.es5.js]: Flipping this makes more sense

```js
if (bindType.indexOf('-') > -1) {
  el.setAttribute(bindType, src);
} else {
  el.style[bindType] = 'url(' + src + ')';
}
```

File: vue-lazyload.es5.js [hunk 3]
```diff
@@ -135,7 +135,12 @@ var vueLazyload = (function (Vue) {
...
+                el.style[bindType] = 'url(' + src + ')';
+            } else {
+                el.setAttribute(bindType, src)
```
Comment (darrynten) [inline vue-lazyload.es5.js]: Missing semicolon

**Comment by ebisbe** [comment]
@hilongjw scrset will be supported or is it right know?

**Comment by hilongjw** [comment]
@ebisbe it's supported at 0.9.5

```javascript
Vue.use(vueLazy, {
    adapter: {
        loaded ({ naturalHeight, naturalWidth, el }) {
            el.setAttribute('data-width', naturalWidth)
            // ...
        },
        loading (listender, Init) {
            console.log('loading')
        },
        error (listender, Init) {
            console.log('error')
        }
    }
})
```

**Comment by ebisbe** [comment]
@hilongjw ok, but how I have to set the object to lazyload?

**Comment by ebisbe** [comment]
@hilongjw Can you tell me how to do it?

**Pull request closed without merging**